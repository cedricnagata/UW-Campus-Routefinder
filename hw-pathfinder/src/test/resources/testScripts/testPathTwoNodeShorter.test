# Test shorter path between two nodes

CreateGraph Graph

AddNode Graph A
AddNode Graph B

AddEdge Graph A B 10.0
AddEdge Graph A B 1.0

FindPath Graph A B