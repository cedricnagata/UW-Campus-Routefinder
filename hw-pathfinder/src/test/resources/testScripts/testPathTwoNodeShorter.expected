# Test shorter path between two nodes

created graph Graph

added node A to Graph
added node B to Graph

added edge 10.000 from A to B in Graph
added edge 1.000 from A to B in Graph

path from A to B:
A to B with weight 1.000
total cost: 1.000